plugin/styleguide-plugin.js

```diff
@@ -79,8 +80,10 @@ StyleguidePlugin.prototype.apply = function apply(compiler) {
     </html>
     `;
 
-    // And emit that HTML template as 'styleguide.html'
-    compilation.assets['styleguide/index.html'] = {
+    const styleguidePath = this.options.basePath || 'styleguide';
```

Comment:
I'm not sure that'll work, because we use routing and I'm pretty sure that needs an `index.html`. We do use hashHistory though, so it might work...